src/SFA.DAS.Provider.PR.Web/Models/EmployerDetailsViewModel.cs

```diff
@@ -0,0 +1,163 @@
+﻿using SFA.DAS.Provider.PR.Domain.OuterApi.Responses;
+
+namespace SFA.DAS.Provider.PR.Web.Models;
+public class EmployerDetailsViewModel
+{
+    public const string NoPermissionText = "No";
```

Comment:
these values are available in SetPermissions.cs in class SetPermissionsText, consider using those instead.